atd-vzd/schema/council_districts.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE public.council_districts(
+    id SERIAL PRIMARY KEY,
+    council_district integer NOT NULL unique,
+    geometry geometry(MultiPolygon, 4326) NOT NULL
```

Comment:
@roseeichelmann it would be a nice touch to add an `updated_at` column (or whatever naming convention y'all use in VZ) to capture the timestamp when the record was created in the db. you could just default that column to `now()`.